ClientApp/src/app/components/miscellaneous/task-overview/task-overview.component.html

```diff
@@ -0,0 +1,54 @@
+<div
+  id="task-overview"
+  pDraggable
```

Comment:
I probably missed this from the demo, but what does the draggable attribute do here? From what I'm reading, don't you also need a droppable somewhere?

Reply:
I'm doing the full drag and drop implementation in a different ticket and that'll require a lot more work, I just added it in here for fun to see what it would look like, especially since I have that draggable icon as part of the task overview anyways.

Reply:
Gotcha, sounds good